### PR TITLE
Add Peak Design

### PR DESCRIPTION
**Pull request by forsgren**
peakdesign

**Issue:** https://github.com/simple-icons/simple-icons/issues/6503
**Alexa rank:** 39,205
  

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
- Hex value picked from within the logo SVG file (https://cdn.shopify.com/s/files/1/2986/1172/t/6/assets/headerlogo-10.svg) found in the header of the official site https://www.peakdesign.com/

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -6802,6 +6802,11 @@
             "hex": "556DB3",
             "source": "https://www.pcgamingwiki.com/wiki/Home"
         },
+        {
+            "title": "Peak Design",
+            "hex": "1C1B1C",
+            "source": "https://cdn.shopify.com/s/files/1/2986/1172/t/6/assets/headerlogo-10.svg"
```
Comment (service-paradis) [inline _data/simple-icons.json:6808]: Since the SVG can be fond in their website header, you can use https://www.peakdesign.com/ only as source

**Review by service-paradis — CHANGES_REQUESTED** [review]
The icon is perfect 💯 Thank you @forsgren for your contribution.

I made a minor comment about the source. Once the change is made, it is ready to merge!

File: _data/simple-icons.json (hunk 1 above)
Comment (forsgren) [reply _data/simple-icons.json]: @service-paradis I've updated the source now 🙂

**Comment by jorgeamadosoria** [comment]
100% match. LGTM

**Pull request merged**